pkg/parsers/kernel/uname_solaris.go

```diff
@@ -0,0 +1,14 @@
+package kernel
+
+import (
+	"golang.org/x/sys/unix"
+)
+
+func uname() (*unix.Utsname, error) {
+	uts := &unix.Utsname{}
+
+	if err := unix.Uname(uts); err != nil {
```

Comment:
Wouldn't this work on all `unix` system ; and thus be more appropriate in a `uname_unix.go` file ?